.github/mergify.yml

```diff
@@ -18,6 +18,7 @@ queue_rules:
       - check-success=test-suite (nautilus)
       - check-success=test-suite (octopus)
       - check-success=test-suite (pacific)
+      - check-success=test-suite (reef)
```

Comment:
Sorry to ask again.

Why didn't we have the configuration to check for **quincy** all this while?

Reply:
Huh. I didn't notice that it was not present. I assume we (I) simply forgot to add it.  I'll add it to the list in this PR.